README

```diff
@@ -16,14 +16,16 @@ Usage
 =====
 
 After installing light-locker, it will auto start along your session and you will be able to lock your session with "light-locker-command -l".
-This will redirect you to VT8 (assuming that your open session was on VT7 and is now kept safe by light-locker) and present LightDM's greeter for unlocking your session again.
+This will redirect you to VT8 (assuming that your open session was on VT7 and is now kept safe by light-locker) and presents LightDM's greeter for unlocking your session.
```

Comment:
«present» was correct, the sentence here is «this will redirect [...] and present»